debug_browser_freeze.py

```
@@ -0,0 +1,112 @@
+# HTMLコンテンツの解析
+response = requests.get("http://localhost:5001/scenarios")
+if response.status_code == 200:
+    html = response.text
+    
+    # JavaScript関連の要素を確認
+    js_files = []
+    import re
+    
+    # scriptタグを抽出
+    script_tags = re.findall(r'<script[^>]*src="([^"]+)"', html)
+    print(f"\n読み込まれているJavaScriptファイル:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)